### DISCUSSION TITLE
v1.36.0

### DISCUSSION THREAD

**Discussion by mikecao** [umami-software/umami#1335] [General]
## Features
- UTM sources are now available in the More details section
![image](https://user-images.githubusercontent.com/477975/180622946-2898446f-63d6-4832-808e-6f9ec224dd84.png)
- Initial work has started on Clickhouse support. We will have full support very soon

## Updates
- Fixed issue with event queries passing the wrong parameters
- Fix bounces/time display for MySQL
- Language translation updates

A huge thanks to all the contributors on this release! @Lemonawa @cywio @Maxime-J @briancao 

<hr /><em>This discussion was created from the release <a href='https://github.com/umami-software/umami/releases/tag/v1.36.0'>v1.36.0</a>.</em>

**Comment by AliMickey** [comment]
Custom api endpoint var still not working.

Can someone please confirm if they have this working?

**Reply by mikecao** [reply]
It needs to be defined during the build process. See https://umami.is/docs/environment-variables

**Reply by AliMickey** [reply]
I get that, but then how am I meant to define it during build with the docker image pulled from github.

**Reply by mikecao** [reply]
You have to build your own using `docker build`.

**Reply by AliMickey** [reply]
> 

That ruins the whole point of using a pre-built image for its ease.

EDIT: I just tried browsing to the custom /api/** endpoint i defined and it resolves fine (not a 404, but an invalid request) similar to /api/collect. So the route is working, is updating the variable in the script the problem? Why is building a custom image needed?

**Comment by razinj** [comment]
Just wanted to say thank you for the UTM section, been waiting for it for some time, thanks for efforts.

**Comment by ayewo** [comment]
@mikecao I'd just like to say thank you for releasing and maintaining this project 👍!

This bit caught my attention in the release notes:
> Initial work has started on Clickhouse support. We will have full support very soon

One or the reasons I picked Umami over Plausible is that deployment is fairly straightforward due to its architectural simplicity. As a bonus, the app is not [resource-intensive](https://github.com/umami-software/umami/discussions/909#discussioncomment-1917527), especially for low traffic websites, which is what I use Umami for.

I remember you mentioned that adding support for ClickHouse would require significant work: https://github.com/umami-software/umami/discussions/909#discussioncomment-1917525 so I (wrongly?) assumed deployment of Umami would stay *simple* 😀 since you are constrained by Prisma's supported RDBMS.

With the work to support ClickHouse, will this architectural simplicity be preserved?

Thanks!

**Reply by mikecao** [reply]
Unlike Plausible we are an open source first product, not a business product made open source. Clickhouse will be optional for anyone willing to go through the trouble of setting it up. We will never drop support for MySQL or PostgreSQL.

**Reply by ayewo** [reply]
Awesome!
